notebooks/data-sources/TestGrid/metrics/number_of_flakes.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>take out "and vice versa" its a little confusing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Sure</p>